modules/my_health/spec/request/v1/message_drafts_request_spec.rb

```diff
@@ -13,120 +13,168 @@
+          expect(response).to have_http_status(:created)
         end
 
-        expect(response).to be_successful
-        expect(response.body).to be_a(String)
-        expect(response).to match_camelized_response_schema('message')
-        expect(response).to have_http_status(:created)
+        it 'responds to PUT #update' do
+          VCR.use_cassette('sm_client/message_drafts/updates_a_draft') do
+            params[:subject] = 'Updated Subject'
+            params[:id] = created_draft_id
+
+            put "/my_health/v1/messaging/message_drafts/#{created_draft_id}", params:
+          end
+
+          expect(response).to be_successful
+          expect(response).to have_http_status(:no_content)
+        end
       end
 
-      it 'responds to PUT #update' do
-        VCR.use_cassette('sm_client/message_drafts/updates_a_draft') do
-          params[:subject] = 'Updated Subject'
-          params[:id] = created_draft_id
+      describe 'reply drafts' do
+        let(:params) { { message_draft: draft.slice(:body) } }
+
+        it 'responds to POST #create' do
+          VCR.use_cassette('sm_client/message_drafts/creates_a_draft_reply') do
+            post "/my_health/v1/messaging/message_drafts/#{reply_id}/replydraft", params:
+          end
 
-          put "/my_health/v1/messaging/message_drafts/#{created_draft_id}", params:
+          expect(response).to be_successful
+          expect(response.body).to be_a(String)
+          expect(response).to match_response_schema('message')
+          expect(response).to have_http_status(:created)
         end
 
-        expect(response).to be_successful
-        expect(response).to have_http_status(:no_content)
+        it 'responds to POST #create when camel-inflected' do
+          VCR.use_cassette('sm_client/message_drafts/creates_a_draft_reply') do
+            post "/my_health/v1/messaging/message_drafts/#{reply_id}/replydraft", params:,
+                                                                                  headers: inflection_header
+          end
+
+          expect(response).to be_successful
+          expect(response.body).to be_a(String)
+          expect(response).to match_camelized_response_schema('message')
+          expect(response).to have_http_status(:created)
+        end
+
+        it 'responds to PUT #update' do
+          VCR.use_cassette('sm_client/message_drafts/updates_a_draft_reply') do
+            params[:body] = 'Updated Body'
+            params[:id] = created_draft_reply_id
+            put "/my_health/v1/messaging/message_drafts/#{reply_id}/replydraft/#{created_draft_reply_id}", params:
+          end
+
+          expect(response).to be_successful
+          expect(response).to have_http_status(:no_content)
+        end
       end
     end
+  end
 
-    describe 'reply drafts' do
-      let(:params) { { message_draft: draft.slice(:body) } }
+  context 'when legacy sm policy' do
+    before do
+      Flipper.disable(:mhv_sm_session_policy)
```

Comment:
same here, is you `allow(Flipper)...` you won't have the possibility of running into flaky specs